stdpopsim/catalog/CaeEle/genetic_maps.py

```diff
@@ -0,0 +1,32 @@
+import stdpopsim
+
+_species = stdpopsim.get_species("CaeEle")
+
+_gm = stdpopsim.GeneticMap(
+    species=_species,
+    id="RockmanRIAIL",
+    description="Genetic map from recombinant inbred advanced intercross lines",
+    long_description="""
+        The authors genotyped 1454 nuclear SNP markers in 236 recombinant
+        inbred advanced intercross lines (RIAILs).
+        The genetic distances were estimated in r/qtl using the Haldane map
+        function, treating observed recombination fractions as though they had
+        been observed in a backcross. The marker density is sufficiently high
+        that the exact form of map function employed has little effect on
+        estimated genetic distances. The tip domains of each chromosome were
+        defined by including all markers between the chromosome ends and the
+        first recombination breakpoint observed in the RIAILs.
+        """,
+    url=("http://sesame.uoregon.edu/~ateterina/" "rockman2009_maps.tgz"),
+    sha256="ef0efa0aec3aa8fcd9800830d83d80bb2525788cf7b28105b7292072fedad8fb",
+    file_pattern="C.elegans.Rockman.Kruglyak.2009.{id}.hapmap.txt",
```

Comment:
It looks like the genetic map files in your tarball are inside a `genetic_map` subdirectory. So this needs to be part of the `file_pattern` too, or stdpopsim won't be able to find the files.
```suggestion
    file_pattern="genetic_map/C.elegans.Rockman.Kruglyak.2009.{id}.hapmap.txt",
```